docs/developer/payments/refunds.mdx

```diff
@@ -204,3 +211,78 @@ mutation {
+
+  overchargedAmount = processedAmount - order.total
+  ```
+
+Below is an example order with multiple transactions and `totalRemainingGrant` at each stage:
+
+1. An order has a total of 100 USD and two `TransactionItem`s: one with a `chargedAmount` of 100 USD and the second with `chargedAmount` of 60 USD.
+   The `totalBalance` would be 60, as `chargedAmount` - `order.total` results in 60. The `chargeStatus` is `OVERCHARGED`.
+2. Adding a granted refund of 10 USD would increase in a `totalBalance` of 10.
+   The `totalRemainingGrant` becomes 10, as the `totalRefundedAmount` is 0.
+3. The second transaction is refunded for an amount of 50 USD.
+   After this operation, the `totalBalance` is 20, and the order remains `OVERCHARGED`. The `totalChargedAmount` decreases, and the `totalRefundedAmount` increases.
+   The `totalRemainingGrant` remains 10 because the `overchargedAmount` still exceeds the original order.total.
+
+   ```python
+   overchargedAmount = totalChargedAmount + totalRefundedAmount - order.total = 110 + 50 - 100 = 60
+
+   totalRefunded = max(totalGrantedRefund - (totalRefunded - overchargedAmount), 0) = max(10 - max(50 - 60, 0)) = 10 - 0 = 10
+   ```
+
+4. In the next step the first transaction was refunded for 15 USD. This brings the total below the `order.total` value, reducing the `totalRemainingGrant`.
+5. Finally, the last refund of 5 USD is processed, resulting in a `totalBalance` of 0. The `chargeStatus` is `FULL`, and the `totalRemainingGrant` is 0.
+
+| Step Nr | total | totalBalance | authorizeStatus | chargeStatus | totalChargedAmount | totalRefundedAmount | orderGrantedRefund.amount | totalRemainingGrant.amount |
+| ------- | ----- | ------------ | --------------- | ------------ | ------------------ | ------------------- | ------------------------- | -------------------------- |
+| 1       | 100   | 60           | FULL            | OVERCHARGED  | 160                | 0                   | 0                         | 0                          |
+| 2       | 100   | 70           | FULL            | OVERCHARGED  | 160                | 0                   | 10                        | 10                         |
+| 3       | 100   | 20           | FULL            | OVERCHARGED  | 110                | 50                  | 10                        | 10                         |
+| 4       | 100   | 5            | FULL            | OVERCHARGED  | 95                 | 65                  | 10                        | 5                          |
+| 5       | 100   | 0            | FULL            | FULL         | 90                 | 70                  | 10                        | 0                          |
+
+> **Note:**
+>
+> - `totalChargedAmount` - is the sum of charged amounts from all transactions
+> - `totalRefundedAmount` - is the sum of all refunded or pending refunds amount from all transactions
+
+:::info
+
+- Pending amounts are treated as they were processed, so are included in all processed amounts.
```

Comment:
Maybe worth emphasizing, that this `info` is related to calculating the granted refund remaining amount. I mean, for example order has `totalCharged` field which doesn't include the pendingChargedAmount. Same for authorizedAmount. So maybe it would be good to explicitly write that we are talking about granted refund (yes, I am aware that we are in refund chapter, but still I was confused for a few seconds). 
It should be enough to add something like: `In case of calculating total remaining grant:`

Reply:
Updated